src/patients/util/patient-name-util.ts

```diff
@@ -19,7 +19,15 @@ export function getPatientName(givenName?: string, familyName?: string, suffix?:
   return name.trim()
 }
 
-export function getPatientFullName(patient: Patient): string {
+export const getPatientCode = (p: Patient | undefined): string => {
```

Comment:
With the addition of this function, this file should be renamed to `patient-util.ts` or we should create a new file `patient-code-util` where this function should live.